static/app/utils/requestError/requestError.tsx

```diff
@@ -2,14 +2,17 @@ import {ResponseMeta} from 'sentry/api';
 
 import {sanitizePath} from './sanitizePath';
 
+interface ErrorOptionsObject {
+  cause: Error;
+}
 export default class RequestError extends Error {
   responseText?: string;
   responseJSON?: any;
   status?: number;
   statusText?: string;
 
-  constructor(method: string | undefined, path: string) {
-    super(`${method || 'GET'} ${sanitizePath(path)}`);
+  constructor(method: string | undefined, path: string, options: ErrorOptionsObject) {
+    super(`${method || 'GET'} "${sanitizePath(path)}"`, options);
```

Comment:
Had to quote this because of a [bug in our SDK](https://github.com/getsentry/sentry-javascript/issues/5498)